### PR TITLE
add reuseble components

### PR DESCRIPTION
**Pull request by MykhailoKutsyi**
Hello, there

Connected  owner profile page.

App wrapped in a container with breaking points.

Added reusable component:
- Button, styled with styled-components.

For now, this page :

image

### CODE REVIEW

File: libs/components/src/lib/container/container.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { ContainerWrapper } from './styles';
+
+/* eslint-disable-next-line */
```
Comment (lyudad) [inline libs/components/src/lib/container/container.tsx]: why so many eslint disable, we create the rules to follow them

Comment (MykhailoKutsyi) [reply libs/components/src/lib/container/container.tsx]: In this case - dont need there, I have props
it came from created component, just missed out it

**Comment by MykhailoKutsyi** [comment]
Hello
added JobPostPage, added JobPostForm 

For now, have form with validation. Connect Selects to Form with Controller from react-hook-form
Styles would be updated.

And also make reuseble component Icon (return svg). Which help more easy include nice svg to project.

**Comment by MykhailoKutsyi** [comment]
When data entered and pressed submit, object collected all data and shown at console, for now, temporary)
Screen 
image

File: libs/components/src/lib/JobPostForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Form, Select } from 'antd';
+
+import Icon from '../assets/Icon';
+import { StyledButton } from '../button/styles';
+import { multipleSelectOption, options } from '../constants/options';
```
Comment (lyudad) [inline libs/components/src/lib/JobPostForm/index.tsx]: absolute path

File: libs/components/src/lib/JobPostForm/styles.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+export const Input = styled.input`
+  margin-left: 20px;
+  height: 30px;
+  outline: none;
+  border: none;
+
+  ::placeholder {
+    font-size: 18px;
+    line-height: 32px;
+    color: #2d2d2d;
```
Comment (lyudad) [inline libs/components/src/lib/JobPostForm/styles.ts:50]: Снимок экрана 2022-11-15 в 20 29 24

**Pull request merged**